build/wix/README.md

```diff
@@ -12,4 +12,11 @@ For example, to build 64 bits package:
 .\build.bat 64
 
 or to build 32 bits package:
-.\build.bat 32
\ No newline at end of file
+.\build.bat 32
+
+To test your multilingual package
+=================================
+
+`msiexec /i mixxx-full-name.msi ProductLanguage=xxxx`
+
+where xxxx is a language code taken from http://www.science.co.il/Language/Locale-codes.asp and available as a translation in build/wix/Localization
```

Comment:
xxxx should be taken from the "LCID decimal" column.
This is 1033 for american english, 1031 for German and 1036 for French and so on.